### PR TITLE
Contextmenu open's only the context menu for the item under the curso…

### PR DESCRIPTION
**Pull request by FelixJoehnk**
As in Issue #196 I also found it confusing that the context menu opens for the selected items and not for the items currently under the courser. This should solve this issue by simply checking if the item was selected or not. Also, it currently doesn't do anything if the item under the cursor is not selectable, but this should never be the case since the context menu should only open something under the cursor from my point of view.

best Regards

### CODE REVIEW

File: packages/sprotty/src/features/context-menu/mouse-listener.ts [hunk 1]
```diff
@@ -46,14 +47,28 @@ export class ContextMenuMouseListener extends MouseListener {
...
-        const menuItems = await this.menuProvider.getItems(target.root, mousePosition);
-        menuService.show(menuItems, mousePosition, restoreSelection);
+
+        if(isSelectable(target)){
+            if(isSelected(target)){
+                const menuItems = await this.menuProvider.getItems(target.root, mousePosition);
+                menuService.show(menuItems, mousePosition, restoreSelection);
+            }else{
+                const options = {selectedElementsIDs: [id], deselectedElementsIDs: Array.from(root.index.all().filter(isSelected), (val) => {return val.id;})};
+                this.actionDispatcher.dispatch(SelectAction.create(options)).then(() => {
```
Comment (spoenemann) [inline packages/sprotty/src/features/context-menu/mouse-listener.ts:66]: The method is `async`, so you can use `await` here instead of a callback chain.

**Review by spoenemann — COMMENTED** [review]
Thanks!

Could you fix the eslint warnings (`yarn lint`)?
```
/workspace/sprotty/packages/sprotty/src/features/context-menu/mouse-listener.ts
  60:9   warning  Expected space(s) after "if"     keyword-spacing
  61:13  warning  Expected space(s) after "if"     keyword-spacing
  64:14  warning  Expected space(s) before "else"  keyword-spacing
  64:14  warning  Expected space(s) after "else"   keyword-spacing
```

File: packages/sprotty/src/features/context-menu/mouse-listener.ts (hunk 1 above)
Comment (FelixJoehnk) [reply packages/sprotty/src/features/context-menu/mouse-listener.ts]: Hey thanks for the reply. Should all be fine now. 
Best Regards

**Review by spoenemann — APPROVED** [review]
:+1:

**Pull request merged**